data-serving/data-service/schemas/cases.schema.json

```diff
@@ -275,7 +275,12 @@
               }
             },
             "purpose": {
-              "bsonType": "string"
+              "enum": [
+                "Business",
+                "Leisure",
+                "Family",
```

Comment:
What's the distinction between family/leisure?

Reply:
I believe leisure = vacation, and family is visiting family. I believe some people do not consider the latter to be leisure :D